jcl/src/java.base/share/classes/java/lang/Access.java

```
@@ -502,5 +509,90 @@ public int encodeASCII(char[] sa, int sp, byte[] da, int dp, int len) {
 	}
 /*[ENDIF] JAVA_SPEC_VERSION >= 17 */
 
+/*[IF JAVA_SPEC_VERSION >= 19]*/
+	public Thread currentCarrierThread() {
+		return Thread.currentCarrierThread();
+	}
+
+	public <T> T getCarrierThreadLocal(ThreadLocal<T> local) {
+		return local.getCarrierThreadLocal();
+	}
+
+	public <T> void setCarrierThreadLocal(ThreadLocal<T> local, T value) {
+		local.setCarrierThreadLocal(value);
+	}
+
+	public <V> V executeOnCarrierThread(Callable<V> task) throws Exception {
+		throw new UnsupportedOperationException();
+	}
+
+	public Continuation getContinuation(Thread thread) {
+		return thread.getContinuation();
+	}
+
+	public void setContinuation(Thread thread, Continuation c) {
+		thread.setContinuation(c);
+	}
+
+	public Object[] extentLocalCache() {
+		return Thread.extentLocalCache();
+	}
+
+	public void setExtentLocalCache(Object[] cache) {
+		Thread.setExtentLocalCache(cache);
+	}
+
+	public Object extentLocalBindings() {
+		return Thread.extentLocalBindings();
+	}
+
+	public void setExtentLocalBindings(Object bindings) {
+		Thread.setExtentLocalBindings(bindings);
+	}
+
+	public StackableScope headStackableScope(Thread thread) {
+		return thread.headStackableScopes();
+	}
+
+	public void setHeadStackableScope(StackableScope scope) {
+		Thread.setHeadStackableScope(scope);
+	}
+
+	public ThreadContainer threadContainer(Thread thread) {
+		return thread.threadContainer();
+	}
+
+	public void start(Thread thread, ThreadContainer container) {
+		thread.start(container);
+	}
+
+	public Thread[] getAllThreads() {
+		return Thread.getAllThreads();
+	}
+
+	public ContinuationScope virtualThreadContinuationScope() {
+		return VirtualThread.continuationScope();
+	}
+
+	public void parkVirtualThread() {
+		VirtualThread.park();
```

you can throw UnsupportedException() for these two